basic1/scripts/script.js

```diff
@@ -0,0 +1,25 @@
+const firstName = document.querySelector("#firstName");
+const lastName = document.querySelector("#lastName");
+const fullName = document.querySelector("#fullName");
+const re = new RegExp("^[A-Za-z]+$");
+/**
+ * This Function Updates the value of the disabled FullName input field.
+ */
+const update = function () {
+  //Checks the firstName format according to the regex. If it fails then it enters the IF block.
+  if (!re.test(firstName.value)) {
+    //If-else structure for lastName. The checks are performed on lastName.
+    if (re.test(lastName.value)) {
```

Comment:
Please mention what are you checking here? Is it checking alphabets or numbers or special chars?
Add a space before starting single line comments. The format will be ->
`// Comment.`